### ISSUE TITLE
Buttons to skip back/forward within a song

### ISSUE THREAD

**Issue by leonardder** [Difegue/Stylophone#73]
First of all, it is probably relevant to say that I'm using the iOS app with VoiceOver as I'm completely blind. I'm pretty amazed by the fact that this Xamarin app is pretty accessible with VoiceOver. However, I'm unable to skip back and forth within a song. May be this feature is there but isn't exposed to VoiceOver somehow?

**Comment by Difegue** [comment]
Wow, I'm impressed that VoiceOver actually works!  
I'll admit I haven't tried it at all... That's probably all thanks to the stock UIKit controls.  

I have some iOS UI work queued up next time I work on a Stylophone release, so I'll try to make sure VoiceOver is enabled for the song progress bar. Thanks!

**Comment by leonardder** [comment]
Thanks, I'll have a look as soon as you've an update out.

**Comment by Difegue** [comment]
👋 The upcoming 2.6 update (currently in App Review) adds missing Voiceover hints for the button to access the fullscreen playback view and the playback slider in said view.  

I hope this'll be enough for accessibility needs - I've tested the app with VoiceOver enabled and was able to use the standard gesture of scrolling up/down on the screen with the slider selected to skip back and forth.

Thanks again for bringing this to my attention!
